actions/copy.js

```diff
@@ -18,7 +19,7 @@ exports.copy = function(from, to, options) {
   to = path.resolve(to);
   options = options || {};
 
-  if (!glob.hasMagic(from)) {
+  if (!Array.isArray(from) && !glob.hasMagic(from)) {
```

Comment:
This check is needed to support copying a single source file to a single destination file, rather than to a specific directory. Removing the check would case the assertion on line 26 to fail for `fs.copy('foo.js', 'bar.js');`.